templates/dns.yml.erb

```diff
@@ -17,7 +16,10 @@
 <%if scope.lookupvar("foreman_proxy::dns_provider") == 'nsupdate_gss' -%>
 :dns_tsig_keytab: <%= scope.lookupvar("foreman_proxy::dns_tsig_keytab") %>
 :dns_tsig_principal: <%= scope.lookupvar("foreman_proxy::dns_tsig_principal") %>
+# dns_key must be disabled if nsupdate_gss is used
+#:dns_key: <%= scope.lookupvar("foreman_proxy::keyfile") %>
```

Comment:
The docs are indeed wrong, they should say to comment it out.  The "false" suggestion probably came from back when we had this bug in 1.5: http://projects.theforeman.org/issues/5675.  We had "false" as a workaround due to the bug in 1.5.0, then fixed the original issue, only to find that keeping it set to false threw the error @smags noted above.  We never did fix it to handle both.